docs/pattern-features.md

```diff
@@ -228,31 +228,6 @@ pattern: $X = 1 + 2 + ...
 foo = 1 + 2 + 3 + 4
 ```
 
-### Deep Expression Operator
```

Comment:
I think this was originally a typo in location. This got lumped in between two "Ellipsis Operator" sub-headers.